app/src/models/progress.ts

```diff
@@ -99,6 +99,12 @@ export interface IRevertProgress extends IProgress {
   kind: 'revert'
 }
 
+/** An object describing the progress of a rebase operation */
+export interface IRebaseProgress {
```

Comment:
I think this should follow the convention of the other progress shapes we've got here. I.e have it extend the `IProgress` interface, state a `kind` and use the fractional value rather than a 0-100 integer value. The `title` should be something like `Rebasing x onto y` or something to that effect.

We should also include it in the Progress union type below.

Reply:
> I.e have it extend the `IProgress` interface, state a `kind` and use the fractional value rather than a 0-100 integer value. The `title` should be something like `Rebasing x onto y` or something to that effect.

I haven't thought about the latter but I think I've already changed `progress` this as part of implementing a working progress bar. I'll pull those changes into this PR.

Reply:
Addressed in 0ec1ffcfe008ad8656337545f030cdaa32821eba